.github/workflows/nightly-release.yml

```diff
@@ -57,3 +57,12 @@ jobs:
     uses: ./.github/workflows/test-apt-packages.yml
     with:
       pkg_type: nightly
+
+  publish-docker-image:
+    needs: test-packages
+    uses: ./.github/workflows/publish-docker-image.yml
```

Comment:
FYI CDN caches are not invalidated atomically in `index-packages.yml`, so we can expect apt installation issues during the image build.

I'm not sure how big of a problem it is, but we'll see in action.